.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # node.js
 #
+node_modules/
```

Comment:
Without those dependencies, should we also remove this?

Reply:
good point, will remove